LearningHub.Nhs.WebUI.AutomatedUiTests/AccessibilityTests/AccessibilityTestsBase.cs

```diff
@@ -0,0 +1,67 @@
+// <copyright file="AccessibilityTestsBase.cs" company="HEE.nhs.uk">
```

Comment:
we need to remove all copyright statements from LH repo (they are not consistent with open-source)